### ISSUE TITLE
Model training fails

### ISSUE THREAD

**Issue by SethCram** [ElementAI/picard#110]
When I run the command 'make train' or 'make eval device=-1' it seems to fail. Here's the resultant output:
![make train fail](https://user-images.githubusercontent.com/86444777/195174092-e822da9c-a127-4f16-9d37-4542d5a76abb.PNG)

I'm running this through WSL2 using an AMD GFX card, but Picard apparently only uses Nvidia GFX cards? I installed the git repo using HTTP instead of the recommended SSH since I didn't have a key. 

Please, let me know if my setup is incorrect. 

Edit:
It looks like the given error code results when there's insufficient memory. Is the model completely untrainable without 40GB of GPU memory? Or do some settings just need to be changed and more time taken?

**Comment by vaib26** [comment]
Hi @SethCram 
Did you solve the setup issue?

**Comment by SethCram** [comment]
@vaib26 No, I did not. I still can't train, evaluate, or serve the model

**Comment by SethCram** [comment]
@vaib26 I don't have an NVIDIA Graphics card, so i was able to run 'make serve' after changing https://github.com/ServiceNow/picard/blob/6a252386bed6d4233f0f13f4562d8ae8608e7445/configs/serve.json#L19 the device number from 0 to -1.
